examples/nlp/parameter_efficient_finetuning_of_gpt2_with_lora.py

```diff
@@ -427,7 +449,7 @@ def call(self, inputs):
 del loss
 
 # This resets "peak" memory usage to "current" memory usage.
-tf.config.experimental.reset_memory_stats("GPU:0")
+# tf.config.experimental.reset_memory_stats("GPU:0")
```

Comment:
Just delete this line? Not needed now that we are using nvidia directly to monitor GPU memory.